<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>per_data</class>
 <widget class="QMainWindow" name="per_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(223, 32, 32);
image: url(:/new/prefix1/image/menu8.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hm_label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>42</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="yd_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>120</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="data_tw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>651</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="select_cb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>83</y>
      <width>221</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>1001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1002</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1003</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="select_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>84</y>
      <width>51</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
